_rules/form-control-non-empty-accessible-name-e086e5.md

```diff
@@ -97,61 +103,54 @@ _There are currently no assumptions_
 
 #### Passed Example 5
 
-[Explicit role](#explicit-role).
+This element with a `combobox` [role][semantic role] has an [accessible name][] because of its `aria-label` attribute.
```

Comment:
Isn't the first role link redundant?

Reply:
Why would it be?

Reply:
Ignore my comment, I didn't realise that "[role]" was a link to semantic role definition... sorry!